ghost/core/core/server/data/schema/schema.js

```diff
@@ -1067,5 +1067,17 @@ module.exports = {
         collection_id: {type: 'string', maxlength: 24, nullable: false, references: 'collections.id', cascadeDelete: true},
         post_id: {type: 'string', maxlength: 24, nullable: false, references: 'posts.id', cascadeDelete: true},
         sort_order: {type: 'integer', nullable: false, unsigned: true, defaultTo: 0}
+    },
+    recommendations: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        url: {type: 'string', maxlength: 2000, nullable: false},
+        title: {type: 'string', maxlength: 2000, nullable: false},
+        excerpt: {type: 'string', maxlength: 2000, nullable: true},
+        featured_image: {type: 'string', maxlength: 2000, nullable: true},
+        favicon: {type: 'string', maxlength: 2000, nullable: true},
+        reason: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Some of the `maxlength`here seem unnecessary large — but decided to keep it safe and take the same values as in the Mentions table